package.json

```diff
@@ -2,12 +2,33 @@
   "name": "app",
   "private": "true",
   "dependencies": {
-    "@hotwired/stimulus": "^3.2.1",
-    "@hotwired/turbo-rails": "^7.3.0",
-    "webpack": "^5.88.1",
-    "webpack-cli": "^5.1.4"
+    "@reduxjs/toolkit": "^1.9.5",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-redux": "^8.1.1",
+    "react-router-dom": "^6.14.1",
+    "webpack": "^5.88.1"
   },
   "scripts": {
-    "build": "webpack --config webpack.config.js"
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --loader:.js=jsx"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.22.8",
+    "@babel/eslint-parser": "^7.22.7",
+    "@babel/plugin-syntax-jsx": "^7.22.5",
+    "@babel/preset-env": "^7.22.7",
+    "@babel/preset-react": "^7.22.5",
+    "esbuild": "^0.14.38",
+    "esbuild-loader": "^2.18.0",
+    "eslint": "^7.32.0",
```

Comment:
- [ ] Based on the project requirement, you should have `babel-loader` installed and because you do not have it, I get the below error. Kindly fix this

![image](https://github.com/ngeelani48/hello-rails-react/assets/84156714/941d720b-2e9f-436f-a254-aad27c8a43bd)

Reply:
@bobb-Rob Thanks for your time and review.

I have installed it